x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/reducer.ts

```diff
@@ -79,8 +79,10 @@ export const policyDetailsReducer: Reducer<PolicyDetailsState, AppAction> = (
 
     // Did user just enter the Detail page? if so, then set the loading indicator and return new state
     if (isCurrentlyOnDetailsPage && !wasPreviouslyOnDetailsPage) {
-      newState.isLoading = true;
-      return newState;
+      return {
+        ...newState,
+        isLoading: true,
```

Comment:
`newState` isn't easy to mutate because its kinda a mix between immutable and not. Its created by spreading an immutable value, so all of its nonprimivite keys are immutable. We could come up w/ a type for this if we need it